crates/typst-kit/src/package.rs

```diff
@@ -14,6 +15,111 @@ use typst::syntax::package::{
 
 use crate::download::{Downloader, Progress};
 
+struct LockFile {
+    path: PathBuf,
+    tmp_path: String,
```

Comment:
I assume the `tmp` prefix here means that this is not included in the final version?

Reply:
I'm not sure about that yet.

Reply:
Well, it's only used for the temporary log messages and can be reconstructed from the actual `path`.

Reply:
Only in a weird way. Instead of just saving the good parts of path directly in `String`, it would need a separate method that will grab the last few parts from the `PathBuf` with either `.unwrap()`s (or `?`, maybe). I don't think that would more clean or more performant. So I would hold on to it for now. It shouldn't matter if struct will be a few bytes bigger, as this is not a critical part of any program.

So I think it's the best overall method of storing the lock file path for logging. But on the other hand, you would probably want a full path to the lock file, not a cache-dir-relative one, right? Then I can remove it and use `path` directly.